python-fs-providers/onedrive_onedrive-fs/fs-provider.json

```diff
@@ -10,6 +10,13 @@
             "label": "OneDrive connection",
             "type":"PRESET",
             "parameterSetId": "onedrive"
+        },
+        {
+            "name": "shared_folder",
+            "label": "Shared folder root",
+            "description": "To view a folder shared with me (Optional)",
```

Comment:
Not sure about using the pronoun "me" in the UI, I found it a little be confusing. I think the usual way to refer the user is "you". 
I would have say something like "To browse files from a folder shared with you"

Reply:
Done in 9960cc2